hello_bot/helpers/html_helper.py

```diff
@@ -0,0 +1,34 @@
+def html_bold(text):
```

Comment:
do we need this file?

Reply:
Don't know, but feel free to fix that in your code cleanup 😄